.github/settings.yml

```diff
@@ -4,14 +4,14 @@
 
 repository:
   name: ursa
-  description: Hyperledger Ursa is a shared cryptography library
+  description: Hyperledger Ursa has moved to end-of-life status.
```

Comment:
Request change to:

Hyperledger Ursa (a shared cryptographic library) has moved to end-of-life status, with the components of Ursa still in use moved to their relevant Hyperledger projects (AnonCreds, Indy, Aries and Iroha).

Reply:
updated